Tests/Amplitude+Test.m

```diff
@@ -66,5 +110,31 @@ - (void)flushUploads:(void (^)(void))handler {
     });
 }
 
++ (void)cleanUpUserDefaults {
+    NSUserDefaults * userDefaults = [NSUserDefaults standardUserDefaults];
+    NSDictionary * dict = [userDefaults dictionaryRepresentation];
+    for (id key in dict) {
+        [userDefaults removeObjectForKey:key];
+    }
+    [userDefaults synchronize];
```

Comment:
nit: do we need the `synchronize` here?

Reply:
It basically makes all the ops getting executed immediately.

Reply:
Just double checked the document https://developer.apple.com/documentation/foundation/userdefaults/1414005-synchronize.
It says `this method is unnecessary and shouldn't be used.` I will remove this.